<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CfilterTransactionDlg</class>
 <widget class="QDialog" name="CfilterTransactionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="m_filterDateCheckbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter Date</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="m_filterDateStart">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>d MMM yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="m_filterDateEnd">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>d MMM yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="m_filterDateLbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter Date Between</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="m_deptLbl">
      <property name="text">
       <string>Department</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_accountLbl">
      <property name="text">
       <string>Account</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_transTypeLbl">
      <property name="text">
       <string>Transaction Type</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="m_deptBox"/>
    </item>
    <item>
     <widget class="QComboBox" name="m_accountBox"/>
    </item>
    <item>
     <widget class="QComboBox" name="m_transTypeBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
